enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -1,20 +1,13 @@
-> **NOTE**: This SDK is actively being developed. You can expect some changes to the API. The SDK currently only works
-> with a specific version of Metabase.
+> **NOTE**: This SDK is actively being developed. You can expect some changes to the API. The SDK currently only works with a specific version of Metabase.
 
 # Metabase Embedding SDK for React
 
 The Metabase Embedding SDK for React offers a way to integrate Metabase into your application more seamlessly and with
 greater flexibility than using the current interactive embedding offering based on iframes.
 
-<div>
-  <a href="https://www.loom.com/share/b6998692937c4ecaab1af097f2123c6f">
-    <img style="max-width: 300px" src="https://cdn.loom.com/sessions/thumbnails/b6998692937c4ecaab1af097f2123c6f-with-play.gif">
-  </a>
-</div>
+Live demo: https://embedded-analytics-sdk-demo.metabase.com/
```

Comment:
Should we use the shorten version just in case we move it again? Also it looks a bit cooler 😎 

```suggestion
Live demo: https://metaba.se/sdk-demo
```